dataactcore/migrations/versions/51679e7bfe01_merge_dsrs_index_with_master.py

```diff
@@ -0,0 +1,36 @@
+"""merge fsrs index with master
+
+Revision ID: 51679e7bfe01
+Revises: d074fd492181, dcbb5afa125e
+Create Date: 2016-09-20 14:46:56.470399
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '51679e7bfe01'
+down_revision = ('d074fd492181', 'dcbb5afa125e')
```

Comment:
oooh, cool--I've been handling alembic conflicts manually...good to know about this.